### DISCUSSION TITLE
Weighted poly's

### DISCUSSION THREAD

**Discussion by alilee** [vleue/polyanya#15] [Q&A] [answered]
Does Polyanya extend to optimal traversal (rather than shortest distance) when "travel cost" is not uniform between poly's? eg. take road around forest for speed.

**Comment by mockersf** [comment, marked as answer]
No, cost is always uniform in Polyanya.
I'm planning to look into how to adapt for that, but no promise if it's possible to keep Polyanya guarantees or when that could happen.

**Reply by alilee** [reply]
Thanks!

**Comment by Utope** [comment]
Any progress on this? I see Scale but that doesn't seem to be adding to cost the way I expect it to.
